src/compiler/diagnosticMessages.json

```diff
@@ -3391,7 +3391,7 @@
         "category": "Error",
         "code": 2791
     },
-    "Cannot find module '{0}'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?": {
+    "Cannot find module '{0}'. Did you mean to set the 'moduleResolution' option to 'nodenext', or to add aliases to the 'paths' option?": {

```

Comment:
I also meant to leave some commentary on this change. This message was written when there were only two module resolution options, and only one worked with non-relative specifiers. Now that there’s `node16`, `nodenext`, and `bundler`, it’s a bit harder to infer what someone on `classic` should be on instead. I picked `nodenext`, but we could revisit this message in the future if we need to.